packages/components/src/components/dropdown-select/dropdown-select.tsx

```diff
@@ -404,9 +406,11 @@ export class DropdownSelect {
       <Host>
         <div part={this.getBasePartMap()}>
           <div part="combobox-container">
-            <label id={`${this.comboboxId}-label`} part="label">
-              {this.label}
-            </label>
+            {!this.hideLabelVisually && (
+              <label id={`${this.comboboxId}-label`} part="label">
+                {this.label}
+              </label>
+            )}
```

Comment:
We shouldn't remove the label completely, it has to remain there for accessibility reasons. So we need to hide it only visually while keeping it in the markup. The old `scale-dropdown` does this, in case it helps.

Reply:
ok Thanks for the input, I will do the change.